src/components/sidebar-toggle-button/README.md

```diff
@@ -0,0 +1,12 @@
+### Examples
+**Open**
+```js
+const isOpen = true;
+<SidebarToggleButton isOpen={isOpen}/>
+```
+**Closed**
+```js
+const isOpen = true;
```

Comment:
Should this be `const isOpen = false;`? Could also be done inline without the variable.